libraries/AP_Scripting/lua/src/luaconf.h

```diff
@@ -207,7 +208,7 @@
 #else			/* }{ */
 
 #define LUA_ROOT	"/usr/local/"
-#define LUA_LDIR	LUA_ROOT "share/lua/" LUA_VDIR "/"
+#define LUA_LDIR	SCRIPTING_DIRECTORY "/modules/"
```

Comment:
I can't find where this is used, do we still need this change? I think if we could remove it we could also drop the hal changes?

Reply:
Ah, I found it.